uavcan/equipment/ahrs/1002.PositionAttitude.uavcan

```diff
@@ -0,0 +1,40 @@
+#
+# Inertial data and orientation in body frame with fused location
+#
+
+uavcan.Timestamp timestamp
+
+float64 longitude                   # angular degree
+float64 latitude                    # angular degree
+float32 height_ellipsoid            # meter
+float32 height_msl                  # meter
+float32 height_baro                 # meter
+float32 ground_distance             # meter (with radar altimeter or LIDAR)
+
+float16[4] orientation_xyzw         # rotation quaternion, zero yaw at north
+
+#
+# Column order:
+#   longitude       [meter^2]
+#   latitude        [meter^2]
+#   height (worse)  [meter^2]
+#   roll angle      [radian^2]
+#   pitch angle     [radian^2]
+#   yaw angle       [radian^2]
+#
+float16 pose_covariance[<=36]
+
+float32[3] linear_velocity_ned      # meter/second
+float16[3] angular_velocity         # rad/sec
+float16[3] linear_acceleration      # m/s^2
```

Comment:
I put it here not for fusion reasons, but as you stated later as a filtered solution to be used right away for logging simple things like uncoordinated turn condition, etc.